content/intro-to-storybook/vue/fr/conclusion.md

```diff
@@ -0,0 +1,34 @@
+---
+title: 'Conclusion'
+description: 'Mettez toutes vos connaissances ensemble et apprenez plus de techniques Storybook'
+---
+
+Félicitations ! Vous avez créé votre première interface utilisateur avec Storybook. En cours de route, vous avez appris comment construire, composer, tester et déployer des composants graphiques. Si vous avez suivi, votre projet et instance Storybook déployée devraient ressembler à ceci :
```

Comment:
I suggest a small change for this translation:

Félicitations ! Vous avez créé votre première interface utilisateur avec Storybook. En cours de route, vous avez appris **à** construire, composer, tester et déployer des composants graphiques. Si vous avez bien suivi, votre projet et **votre** instance Storybook **une fois** déployée devraient ressembler à ceci :

Reply:
Ok effectivement, j'ajoute ça de suite